app/assets/stylesheets/components/_filter-summary.scss

```diff
@@ -0,0 +1,60 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-c-filter-summary__remove-filters {
+  list-style-type: none;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.app-c-filter-summary__remove-filter {
+  border: 1px solid $govuk-border-colour;
+  border-radius: govuk-em(5, 16px);
+  padding: govuk-spacing(2);
+  margin: 0 govuk-spacing(2) govuk-spacing(1) 0;
+  text-decoration: none;
+  display: inline-block;
+  color: $govuk-text-colour;
+  background-color: govuk-colour("light-grey");
+  @include govuk-font(16);
+
+  @include govuk-media-query($from: tablet) {
+    text-wrap: nowrap;
+  }
+
+  &:focus,
+  &:focus-visible {
```

Comment:
Do we need to target `:focus-visible` separately now it has the same styles at `:focus`?